### DISCUSSION TITLE
I just made telegram bot for this app

### DISCUSSION THREAD

**Discussion by yellowgg2** [usememos/memos#727] [Q&A]
I just created [memos-bot](https://github.com/yellowgg2/memos-telegram-bot).

I am just wondering if I have to make PR to change README by myself or someone can do this for me.

Also, is there a plan to add more REST APIs?

Thanks for the great app!

**Comment by Zeng1998** [comment]
> I am just wondering if I have to make PR to change README by myself or someone can do this for me.

You can make a PR just by yourself.

> Also, is there a plan to add more REST APIs?

Do you mean the docs for the REST APIs?

**Reply by yellowgg2** [reply]
> > I am just wondering if I have to make PR to change README by myself or someone can do this for me.
> 
> 
> 
> You can make a PR just by yourself.
> 
> 
> 
> > Also, is there a plan to add more REST APIs?
> 
> 
> 
> Do you mean the docs for the REST APIs?
> 
> 

I mean there is only one api I can find is create new memo. Are there more APIs undocumented such as searching memo, getting list of tags and etc?

**Reply by Zeng1998** [reply]
> > > I am just wondering if I have to make PR to change README by myself or someone can do this for me.
> > 
> > 
> > You can make a PR just by yourself.
> > > Also, is there a plan to add more REST APIs?
> > 
> > 
> > Do you mean the docs for the REST APIs?
> 
> I mean there is only one api I can find is create new memo. Are there more APIs undocumented such as searching memo, getting list of tags and etc?

Absolutely yes, you can see `server/*.go` for more details.

**Reply by yellowgg2** [reply]
I didn't even think about checking the code directly.

Thank you so much
